MekHQ/src/mekhq/campaign/autoResolve/damageHandler/EntityMustSurvive.java

```diff
@@ -0,0 +1,6 @@
+package mekhq.campaign.autoResolve.damageHandler;
```

Comment:
Similar to my last comment, does this need to be a separate class? Can we merge the two enums, seeing that they have the same entries with identical ords